Solutions/Rapid7InsightVM/Playbooks/readme.md

```diff
@@ -0,0 +1,94 @@
+
+This package includes:
+
+* [Logic Apps custom connector for Rapid7 InsightVM Cloud Integrations API](./Rapid7InsightVMCloudAPIConnector)
+
+
+* These three playbook templates leverage Rapid7 InsightVM custom connector:
+  * [Rapid7InsightVM-EnrichIncidentWithAssetInfo](./Playbooks/Rapid7InsightVM-EnrichIncidentWithAssetInfo)
+  * [Rapid7InsightVM-EnrichVulnerabilityInfo](./Playbooks/Rapid7InsightVM-EnrichVulnerabilityInfo)
+  * [Rapid7InsightVM-RunScan](./Playbooks/Rapid7InsightVM-RunScan)
+
+You can choose to deploy the whole package: connectors + all three playbook templates, or each one seperately from its specific folder.
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FRapid7InsightVM%2FPlaybooks%2Fazuredeploy.json) [![Deploy to Azure](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FRapid7InsightVM%2FPlaybooks%2Fazuredeploy.json)
+
+# Rapid7 InsightVM connectors documentation 
+
+<a name="authentication">
+
+## Authentication
+
+* API Key authentication
+
+<a name="prerequisites">
+
+### Prerequisites in Rapid7 InsightVM
+
+To get Rapid7 InsightVM API key, follow the instructions in the [documentation](https://docs.rapid7.com/insight/managing-platform-api-keys/).
+
+<a name="deployment">
+
+### Deployment instructions 
+
+1. To deploy Custom Connectors and Playbooks, click the Deploy to Azure button. This will launch the ARM Template deployment wizard.
+2. Fill in the required parameters for deploying Custom Connectors and Playbooks
+
+| Parameters | Description |
+|----------------|--------------|
+|**For Connector**|
+|**Insight Platform Region** | The region indicates the geo-location of the Insight Platform. |
+|**For Playbooks**|
+|**Rapid7InsightVM-EnrichIncidentWithAssetInfo Playbook Name** | Name of the Playbook |
+|**Rapid7InsightVM-EnrichVulnerabilityInfo Playbook Name** | Name of the Playbook |
+|**Rapid7InsightVM-RunScan Playbook Name** | Name of the Playbook |
+|**TeamsGroupId** | Id of the Teams Group where the adaptive card will be posted (for playbook Rapid7InsightVM-RunScan). |
+|**TeamsChannelId** | Id of the Teams Channel where the adaptive card will be posted (for playbook Rapid7InsightVM-RunScan). |
+
+<br>
+<a name="postdeployment">
+
+### Post-Deployment instructions
+
+#### a. Authorize connections
+
+Once deployment is complete, authorize each connection.
+
+1. Click the Azure Sentinel connection resource
+2. Click edit API connection
+3. Click Authorize
+4. Sign in
+5. Click Save
+6. Repeat steps for other connections
+
+#### b. Configurations in Sentinel
+
+Each Playbook requires a different type of configuration. Check documentation for each Playbook.
+
+<a name="limitations">
+
+## Known Issues and Limitations
```

Comment:
Since this is empty, can we remove it for now.

Reply:
updated